.github/workflows/README.md

```diff
@@ -106,11 +99,14 @@ jobs:
   call-workflow-passing-data:
     uses: bakdata/ci-templates/.github/workflows/helm-release.yaml@main
     with:
-      release-type: "${{ github.event.inputs.release-type }}"
+      release-type: ${{ inputs.release-type }}
       ref: "my-awesome-ref" # (Optional)
       lint-config-path: "my-lint-config.yaml" # (Optional)
-      helm-version: "v3.4.0" # (Optional)
+      helm-version: "v3.10.1" # (Optional)
       python-version: "3.8" # (Optional)
```

Comment:
We do not support a python version anymore right?

Reply:
indeed, I will remove it